lib/compilers/ppci.ts

```diff
@@ -51,10 +53,10 @@ export class PPCICompiler extends BaseCompiler {
                 const pythonPath = matches[1];
                 const ppciName = `ppci.${matches[3].replace('/', '.')}`;
                 options.env = {PYTHONPATH: pythonPath};
-                let python_args = ['-m', ppciName].concat(args);
+                const python_args = ['-m', ppciName].concat(args);
                 return exec.execute(python, python_args, options);
             }
-            logger.error(`Invalid ppci path ${compiler}`);
+            assert(false, `Invalid ppci path ${compiler}`);
```

Comment:
I think an assert here is appropriate. Something to kill the code path is needed otherwise TS isn't happy that undefined is being returned.

Reply:
How is assert(false) better than `throw new Error()` in this case?

Reply:
I think that'd be more or less equivalent. `assert` does ultimately throw a new Error internally. So it's just whatever extra diagnostics `assert` adds in.

Reply:
Got it. Maybe we should make a fatalError(); that makes it clear? (I just have a vague dislike of `assert(false)` but that's mainly C PTSD..:)